### ISSUE TITLE
blocking call to a view blocks the first action execution

### ISSUE THREAD

**Issue by aludwiko** [lightbend/kalix-jvm-sdk#1887]
Reported by Shishir https://github.com/ceecer1/view-query-wiped-out-test/issues/3

It's easy to reproduce the problem by creating an action that will block when calling a view (I suppose any Kalix component):
```
public class ServiceMetaDataActionImpl extends AbstractServiceMetaDataAction {

  private static Logger logger = LoggerFactory.getLogger(ServiceMetaDataActionImpl.class);

  public ServiceMetaDataActionImpl(ActionCreationContext creationContext) {}

  @Override
  public Effect<ProductDomain.ServiceMetaDataState> getServiceMetaData(Empty empty) {

    try {
      CompletionStage<ProductDomain.ServiceMetaDataState> responseStage =
        components().serviceMetaDataMappingViewImpl().getServiceMetaData(Any.newBuilder().build()).execute();

      ProductDomain.ServiceMetaDataState state = responseStage.toCompletableFuture().get(20, TimeUnit.SECONDS);
      return effects().reply(state);
    } catch (Exception ex) {
      ex.printStackTrace();
      logger.info("Error in Get Call--");
    }

    return effects().error("Error processing the request");
  }
}
```

The outcome is that the **first** call to a view is delayed exactly by 20 seconds (blocking future timeout). Logs from the proxy:
```
15:29:39.826 DEBUG kalix.proxy.Serve$ - Received gRPC request [/com.example.action.ServiceMetaDataAction/GetServiceMetaData]
15:30:00.116 DEBUG kalix.proxy.Serve$ - Received gRPC request [/com.example.view.ServiceMetaDataMappingView/GetServiceMetaData]
```
After the first call, the action behaves as expected.

**Comment by johanandren** [comment]
Related existing issue: https://github.com/lightbend/kalix-jvm-sdk/issues/1077

**Comment by johanandren** [comment]
Also related: https://github.com/akka/akka-http/issues/4150 (but I thought we decided to accept the cost of an additional .async for all the Kalix JVM SDK component streams instead of running them in-line)

**Comment by octonato** [comment]
> Why is that view not replying within at most two-three seconds though?

It's worse than that. It seems that the CompletionStage is not scheduled until the timeout in the `get()` happens. We change it to other values and we see it back in the logs in the runtime. If we set a timeout to 7 seconds, we see:
```
15:29:39.826 DEBUG kalix.proxy.Serve$ - Received gRPC request [/com.example.action.ServiceMetaDataAction/GetServiceMetaData]
15:30:06.826 DEBUG kalix.proxy.Serve$ - Received gRPC request [/com.example.view.ServiceMetaDataMappingView/GetServiceMetaData]
```

After that, subsequent calls works immediately. 

So I suspect that we are tweaking the thread pool inside akka-grpc (java api) and this is done lazily. 

I have the impression that the timeout is triggering something that is providing a thread to complete the CompletionStage (so the call is effectively done at that moment), but then the CompletionStage will fail anyway because it already timeout.

To recap, if a timeout of 20 seconds:
* call hit the runtime after 20 seconds
* call fails with timeout on sdk side

**Comment by aludwiko** [comment]
And another one: https://discuss.kalix.io/t/how-to-call-and-get-value-from-view-in-controller/318

**Comment by johanandren** [comment]
Fixed by the virtual threads support https://github.com/lightbend/kalix-runtime/pull/3709
